tests/flow/random_graph.py

```
@@ -0,0 +1,262 @@
+                           random_bool, random_double]
+
+
+def random_array():
+    l = randint(0, 10)
+    arr = []
+    for i in range(0, l):
+        arr.append(random_graph_value(ALL_VALUES_EXCEPT_ARRAY))
+    return arr
+
+def random_point():
+    raise "not implemented"
```

Ruff (0.12.2)

Cannot raise a literal. Did you intend to return it or raise an Exception?

(B016)